Emby.Server.Implementations/IO/ManagedFileSystem.cs

```diff
@@ -50,7 +50,7 @@ public class ManagedFileSystem : IFileSystem
             _isEnvironmentCaseInsensitive = environmentInfo.OperatingSystem == MediaBrowser.Model.System.OperatingSystem.Windows;
         }
 
-        public string DefaultDirectory
+        public virtual string DefaultDirectory
```

Comment:
This `virtual` (and `abstract` too) is implied due it it being a interface method (in CIL). The refactor engine added these, since it is more clear what are interface methods and what are not. This is pure syntactic sugar though. Interface implementation members do not even need public I believe.

The only difference is that child classes of ManagedFileSystem, can now override the virtual members. Since they are now both virtual in C# and CIL. While without the keyword they are not overrideable.

Not sure what the "correct" one is here semantically.